packages/core/src/renderer/createDOMRenderer.ts

```diff
@@ -43,16 +44,17 @@ export function createDOMRenderer(
       // eslint-disable-next-line guard-for-in
       for (const styleBucketName in cssRules) {
         const cssRulesForBucket = cssRules[styleBucketName as StyleBucketName]!;
-        const sheet = getStyleSheetForBucket(
-          styleBucketName as StyleBucketName,
-          target,
-          renderer,
-          options.styleElementAttributes,
-        );
 
         // This is a hot path in rendering styles: ".length" is cached in "l" var to avoid accesses the property
         for (let i = 0, l = cssRulesForBucket.length; i < l; i++) {
-          const ruleCSS = cssRulesForBucket[i];
+          const [ruleCSS, metadata] = normalizeCSSBucketEntry(cssRulesForBucket[i]);
+          const sheet = getStyleSheetForBucket(
```

Comment:
should benchmark this with master to see if more function calls will affect hot path perf

Reply:
I ran 10 benchmark runs on the benchmark app for this branch and main and the numbers seem similar

main: 

![image](https://user-images.githubusercontent.com/20744592/174560654-b0f4b6c4-b6b8-4d4f-98bb-34c52dbf29d3.png)

PR:

![griffel-before](https://user-images.githubusercontent.com/20744592/174560712-e2d056d4-dad7-4819-8fc7-6675bc64c3c8.png)